src/main/java/Daniela_Chihai/com/Endava/SoA5/Java_References/Arrays_Ex1.java

```diff
@@ -0,0 +1,35 @@
+package Daniela_Chihai.com.Endava.SoA5.Java_References;
+
+public class Arrays_Ex1 {
+    public static int numbersMultiplication(int[] a) {      //banana is the method; int[] a{3,8,9} - array of parameters
```

Comment:
Please rename "int[] a" to "int[] array"

Reply:
Done